app/db/models.py

```diff
@@ -1,10 +1,35 @@
-from sqlalchemy import Column, Integer, String, Boolean, DateTime
+from sqlalchemy import Column, Integer, Boolean, String, Numeric, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
 from app.db.database import Base
 
+
 class ParkingSlot(Base):
-    __tablename__ = "parking_slot"
+    __tablename__ = "parking_slots"
 
     id = Column(Integer, primary_key=True, index=True)
     is_occupied = Column(Boolean, default=False)
-    vehicle_number = Column(String(50), nullable=True)
-    entering_time = Column(DateTime, nullable=True)
+
+
+class VehicleRegistration(Base):
+    __tablename__ = "vehicle_registration"
+
+    vehicle_id = Column(Integer, primary_key=True, index=True)
+    license_plate = Column(String(20), unique=True, index=True)
+
+    parking_sessions = relationship("ParkingSession", back_populates="vehicle")
+
+
+class ParkingSession(Base):
+    __tablename__ = "parking_sessions"
+
+    session_id = Column(Integer, primary_key=True, index=True)
+    slot_id = Column(Integer, ForeignKey('parking_slots.id'))
```

Comment:
Add index as it will be used for comparison. It will improve the performance. Both for slot\_id and vehicle\_id